openedx/core/djangoapps/auth_exchange/tests/test_forms.py

```diff
@@ -38,7 +38,6 @@ def _assert_error(self, data, expected_error, expected_error_description):
             form.errors,
             {"error": expected_error, "error_description": expected_error_description}
         )
-        self.assertNotIn("partial_pipeline", self.request.session)
```

Comment:
@UmanShahzad I'm not 100% sure which one is a replacement test - could you either add it (if it's still missing) or point it out for me?